### ISSUE TITLE
Bug 54110519: [React Native Gallery]: Unable to differentiate between selected and unselected menu item present in left navigation pane after applying high contrast aquatic and desert themes.

### ISSUE THREAD

**Issue by Yajur-Grover** [microsoft/react-native-gallery#501]
Build Details:
OS Version: Dev (OS Build 27695.1000)
App: React Native Gallery 
App Version: 1.0.18.0
React Native Windows Version: 0.75.0
AT: High Contrast 

Pre-Requisite:  

Turn on High Contrast Aquatic Theme (Settings->Accessibility->Contrast themes->Aquatic/Dessert theme->Apply)

Repro-Steps:
Launch the React Native Gallery app.
Navigate to hamburger button and expand it.
Now navigate to any button and select it.
Now observe the issue in high contrast themes.
Actual Result:
Unable to differentiate between selected and unselected menu item present in left navigation pane after applying high contrast aquatic and desert themes.

Expected Result:
User should be able to differentiate between selected and unselected menu item present in left navigation pane after applying high contrast aquatic and desert themes.

User impact:

Users with low vision will not be able to differentiate between selected and unselected menu item if it is not visible in High Contrast Themes.

Link to ADO bug: https://dev.azure.com/microsoft/OS/_workitems/edit/54110519
